src/utils/cacheRequest.js

```diff
@@ -57,21 +62,36 @@ const fetchCachedGeneExpressionWork = async (experimentId, timeout, body) => {
   return responseData;
 };
 
-const fetchCachedWork = async (experimentId, timeout, body) => {
-  if (isBrowser) {
-    if (body.name === 'GeneExpression') {
-      return fetchCachedGeneExpressionWork(experimentId, timeout, body);
-    }
-    const key = createObjectHash({ experimentId, body });
-    const data = await cache.get(key);
+const fetchCachedWork = async (experimentId, timeout, body, getState) => {
```

Comment:
For this and `fetchCachedGeneExpressionWork`, could we pass only the pipeline status instead?

Passing the `getState` function is coupling both of these functions with redux and the internal structure of the store.